src/daos-gen/daos-gen.c

```diff
@@ -0,0 +1,302 @@
+#include <stdio.h>
+#include <getopt.h>
+#include <string.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <inttypes.h>
+#include <unistd.h>
+#include <errno.h>
+#include <stdbool.h>
+
+/* for daos */
+#include "mfu_daos.h"
+#include "mpi.h"
+#include "mfu.h"
+
+#include "mfu_errors.h"
+
+#define BUFLEN 80
+#define UUID_STR_LEN 129
+
+static uint32_t obj_id_gen = 1;
+
+/** Print a usage message. */
+void print_usage(void)
+{
+    printf("\n");
+    printf("Usage: daos-gen [options]\n");
+    printf("\n");
+    printf("Options:\n");
+    printf("  -p, --pool               - pool uuid for containers\n");
```

Comment:
@dsikich If we want to include the other options.
```suggestion
    printf("  -p, --pool               - pool uuid for containers\n");
    printf("  -o, --num-objects  - number of objects to generate\n");
    printf("  -k, --keys-per-object - number of keys per object\n");
```